runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -45,7 +45,7 @@ class MaxPool2D final : public TrainingKernelRegistry
   std::unique_ptr<Tensor> _arg_max_index;
 
 public:
-  MaxPool2D(const uint32_t paddingLeft, const uint32_t, const uint32_t, const uint32_t paddingTop,
+  MaxPool2D(const uint32_t paddingLeft, const uint32_t, const uint32_t paddingTop, const uint32_t,
```

Comment:
@ragmani This line was a problem. :smiling_face_with_tear:

Reply:
Are you going to work on it with another PR? Or here?

Reply:
Are you going to work on it at another PR? Or here?

Reply:
I'll make another PR to fix `stirde >1`. 
But I don't know a reason why `stride>1` gets a wrong value. So it might take some time.

Reply:
It's not urgent :)

Reply:
I hope this PR is merged as it is, after getting reviewed.

Reply:
Ah... mnist model of hugging face may have max_pool with `stride>1`, but I'm not sure.

Reply:
@ragmani, @jyoungyun  

`stride > 1` cases also work well! 

Yesterday, I didn't set the epoch properly, that's why the test failed. 
Now test with stride =2 also works well with properlly set epoch.

Reply:
I checked kaggle model with this commit. The loss values ​​of `tensorflow` and `onert_train` were exactly the same. It proves not only this commit but also "stride > 1 cases also work well" is right.
- `tensorflow` result (mse, sgd, learning_rate=1.0f)
```
Epoch 1/5
10/10 [==============================] - 0s 6ms/step - loss: 0.0882 - categorical_accuracy: 0.9000
Epoch 2/5
10/10 [==============================] - 0s 5ms/step - loss: 0.0835 - categorical_accuracy: 1.0000
Epoch 3/5
10/10 [==============================] - 0s 5ms/step - loss: 0.0772 - categorical_accuracy: 1.0000
Epoch 4/5
10/10 [==============================] - 0s 5ms/step - loss: 0.0688 - categorical_accuracy: 1.0000
Epoch 5/5
10/10 [==============================] - 0s 5ms/step - loss: 0.0583 - categorical_accuracy: 1.0000
```

- `onert_train` result (mse, sgd, learning_rate=1.0f)
```
Epoch 1/5 - time: 19.255ms/step - loss: [0] 0.0882
Epoch 2/5 - time: 19.089ms/step - loss: [0] 0.0835
Epoch 3/5 - time: 18.987ms/step - loss: [0] 0.0772
Epoch 4/5 - time: 19.199ms/step - loss: [0] 0.0688
Epoch 5/5 - time: 19.255ms/step - loss: [0] 0.0583
```

Reply:
If possible, it would be helpful to add a `mnist` model in CI to make sure that each PR is not broke the basic training feature.

Reply:
I agree with @zetwhite